components/automate-ui/src/app/pages/job-add/job-add.component.ts

```diff
@@ -93,14 +93,14 @@ export class JobAddComponent implements OnDestroy {
     const nodesGroup = this.fb.group({
       managers: this.fb.array([])
     }, {
-      validator: nodeSelectionRequiredValidator
+      validators: nodeSelectionRequiredValidator
```

Comment:
validator was deprecated as part of the FormBuilder#group deprecation in favor of Abstract control and `validators`
ref: https://angular.io/api/forms/FormBuilder#group